Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -661,6 +661,21 @@ internal sealed partial class NodejsPackage : CommonPackage {
             LogLooseFileAnalysisLevel();
         }
 
+        private VsProjectAnalyzer CreateLooseVsProjectAnalyzer() {
```

Comment:
The two usage scenarios are:
* opening loose files, which now just throws to JSLS
* interactive window